desk/app/activity.hoon

```diff
@@ -644,9 +644,10 @@
   |-
   ?~  stream
     [newest total notified ?~(last ~ `[u.last main main-notified]) children]
-  =/  [[* =event:a] rest=stream:a]  (pop:on-event:a stream)
+  =/  [[=time =event:a] rest=stream:a]  (pop:on-event:a stream)
   =/  volume  (get-volume -.event)
   =?  notified  &(notify.volume notified.event)  &
+  =.  newest  time
   ?.  ?&  unreads.volume
           ::TODO  support other event types
           ?=(?(%dm-post %dm-reply %post %reply) -<.event)
```

Comment:
right any event should potentially affect the recency although we may want to limit it at some point like you said. mostly did this because `chan-init` and `dm-invite` weren't being taken into account, can address in the follow-up